app/routes/git_tokens.py

```
@@ -91,106 +46,296 @@ def mask_token(token: str) -> str:
+        git_label = await GitLabel.create(
+            label=payload.label,
+            user_id=payload.user_id,
+            git_hosting=payload.git_hosting,
+            token_value=encrypted_token,
+            username=user.get("login", ""),
+        )
+
+        return APIResponse.success(
+            message=constants.TOKEN_SAVED_SUCCESSFULLY, data={"id": str(git_label.id)}
+        )
+    except Exception as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)